### PR TITLE
added missing constant for report size truncated

### PR DESCRIPTION
**Pull request by Goirad**

### CODE REVIEW

File: sgx-isa/src/lib.rs [hunk 1]
```diff
@@ -556,6 +556,8 @@ pub struct Report {
 
 impl Report {
     pub const UNPADDED_SIZE: usize = 432;
+    //size without keyid and mac
```
Comment (jethrogb) [inline sgx-isa/src/lib.rs]: Make it a doc comment

File: sgx-isa/src/lib.rs [hunk 2]
```diff
@@ -556,6 +556,8 @@ pub struct Report {
...
+    pub const SIZE_TRUNCATED: usize = 384;
```
Comment (jethrogb) [inline sgx-isa/src/lib.rs]: follow existing naming convention: `TRUNCATED_SIZE`

**Comment by jethrogb** [comment]
bors r-

nevermind need to fix nightly

**Comment by Goirad** [comment]
@jethrogb review ping

**Comment by Goirad** [comment]
@jethrogb sorry, forgot to rebase

**Comment by jethrogb** [comment]
Rebase, why?

bors r+

**Comment by Goirad** [comment]
Over the the nightly fixes commit that just got pulled in

**Comment by jethrogb** [comment]
Yes, but why?

**Pull request merged**